Sources/MapboxSearch/PublicAPI/Common/Models/Language/Language.swift

```diff
@@ -0,0 +1,21 @@
+// Copyright © 2022 Mapbox. All rights reserved.
+
+import Foundation
+
+public struct Language: Equatable {
+    public let languageCode: String
+    
+    /// Country model initializier
+    /// - Parameter languageCode: Permitted values are ISO 639-1 language codes
+    public init?(languageCode: String) {
+        guard let isoCode = ISO639_1(rawValue: languageCode.lowercased()) else {
+            return nil
+        }
+        
+        self.languageCode = isoCode.rawValue
+    }
+    
+    static var `default`: Self {
+        Language(languageCode: "en")!
```

Comment:
Is it possible to get current device language on iOS? in this case device language could be a default value.

Reply:
I've added `en` language as default according to all other types in our SDK, since all of the use `en` as fallback language.
It is make sense to use device language, but it is a bit tricky on iOS and has to be done in a separate PR.